.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@ docs @rancher/mapps
 scripts @rancher/mapps
 CNAME @rancher/mapps
 Makefile @rancher/mapps
-README.md @rancher/mapps
+/README.md @rancher/mapps
```

Comment:
Why not with the rest of file in the root?

Reply:
I'll add it to be safe, but I don't see PRs with files named like the rest of them.